tmt/steps/provision/local.py

```diff
@@ -76,25 +80,32 @@ def execute(self, command, **kwargs):
         # Run the command under the prepared environment
         return self.run(command, env=environment, shell=True, **kwargs)
 
-    def stop(self):
+    def stop(self) -> None:
         """ Stop the guest """
 
         self.debug(f"Doing nothing to stop guest '{self.guest}'.")
 
-    def reboot(self, hard=False, command=None, timeout=None):
+    def reboot(self,
+               hard: bool = False,
+               command: Optional[Union[List[str], str]] = None,
```

Comment:
hmm, I wonder why `command` has here different type that in https://github.com/teemtee/tmt/blob/1920b07682e55c4d58fcbca76011505432e9fd76/tmt/steps/provision/__init__.py#L559?

Reply:
Yep, should be an `Optional[str]`. Likely a leftover from the time when `command` could be both `list` or `str`.